CHANGELOG.md

```diff
@@ -219,34 +54,35 @@ See [Upgrading versions][] for more information about upgrading between PWA Stud
 ### Update dependencies
 
 Open your `package.json` file and update the PWA Studio package dependencies to the versions associated with this release.
-The following table lists the latest versions of each package as of 12.0.0.
+The following table lists the latest versions of each package as of 12.1.0.
 Versions that are in **bold** indicate a version change for this release.
 
 **Note:**
 Your project may not depend on some of the packages listed on this table.
 
 | Package                         | Latest version |
-|---------------------------------|----------------|
+| ------------------------------- | -------------- |
 | `babel-preset-peregrine`        | 1.1.0          |
-| `create-pwa`                    | **2.0.0**      |
-| `upward-security-headers`       | 1.0.4          |
-| `venia-adobe-data-layer`        | 1.0.1          |
+| `create-pwa`                    | **2.0.1**      |
+| `upward-security-headers`       | **1.0.5**      |
+| `venia-adobe-data-layer`        | **1.0.2**      |
 | `venia-sample-backends`         | 0.0.4          |
-| `venia-sample-language-packs`   | 0.0.4          |
-| `venia-sample-payments-checkmo` | 0.0.2          |
-| `pagebuilder`                   | **7.0.0**      |
-| `peregrine`                     | **12.0.0**     |
-| `pwa-buildpack`                 | 10.0.0         |
-| `upward-js`                     | **5.2.0**      |
-| `upward-spec`                   | **5.1.0**      |
-| `venia-concept`                 | **12.0.0**     |
-| `venia-ui`                      | **9.0.0**      |
-| `magento2-upward-connector`     | **1.3.0**      |
-| `upward-php`                    | **1.2.0**      |
+| `venia-sample-language-packs`   | **0.0.5**      |
+| `venia-sample-payments-checkmo` | **0.0.3**      |
+| `pagebuilder`                   | **7.0.1**      |
+| `peregrine`                     | **12.1.0**     |
+| `pwa-buildpack`                 | 11.0.0         |
+| `pwa-theme-venia`               | **1.1.0**      |
+| `upward-js`                     | 5.2.0          |
+| `upward-spec`                   | 5.1.0          |
+| `venia-concept`                 | **12.1.0**     |
+| `venia-ui`                      | **9.1.0**      |
+| `magento2-upward-connector`     | 1.3.0          |
+| `upward-php`                    | 1.2.0          |
 
 ### Update template files
```

Comment:
I'm not sure that any of these files were updated for `12.1.0`.

Reply:
Got it. The non-bolded packages in the list indicate that they have not been updated for this release. And the two bolded ones, `venia-concept **12.1.0**` and ` venia-ui  | **9.1.0**` were alpha last time I looked in the release/12.1 branch. I'll recheck.

Reply:
@dpatil-magento Can you verify the package updates above? Currently, all the ones marked new are `beta-1`. Will they be released that way?

Reply:
Sorry, GitHub highlighted a block there but I meant to only comment on line 247. Looks like you took these out anyway 👍